libs/saveFiles.spec.js

```diff
@@ -122,4 +130,37 @@ describe('saveFiles', function() {
       })
     })
   }
+
+  const billWithoutFilename = [{
+    amount: 62.93,
+    date: '2018-03-03T23:00:00.000Z',
+    vendor: 'Free Mobile',
+    type: 'phone',
+    filestream: 'mock stream'
+  }]
+  describe('when filestream is used without filename', () => {
+    it('should throw an error', async () => {
+      expect.assertions(2)
+      try {
+        await saveFiles(billWithoutFilename, options)
+      } catch (error) {
+        expect(error).toEqual(new Error('Missing filename property'))
+      }
+      expect(cozyClient.files.create).not.toHaveBeenCalled()
+    })
+  })
+
+  const billWithoutStreamUrlAndRequestOptions = [{
+    amount: 62.93,
+    date: '2018-03-03T23:00:00.000Z',
+    vendor: 'Free Mobile',
+    type: 'phone'
+  }]
+  describe('when filestream is used without filename', () => {
+    it('should throw an error', async () => {
```

Comment:
I think the text of the assertion is not good ? `it('should do nothing')` maybe ?

Reply:
Yes, it is a failed copy/paste, the describe need to be fixed too.